src/components/scenes/FioRequestListScene.js

```diff
@@ -0,0 +1,481 @@
+// @flow
+
+import React, { Component } from 'react'
+import { Alert, Animated, FlatList, Image, Platform, TouchableHighlight, TouchableOpacity, View } from 'react-native'
+import { Actions } from 'react-native-router-flux'
+import slowlog from 'react-native-slowlog'
+
+import checkedIcon from '../../assets/images/createWallet/check_icon_lg.png'
+import invalidIcon from '../../assets/images/createWallet/invalid_icon.png'
+import fioRequestsIcon from '../../assets/images/fio/fio_sent_request.png'
+import * as Constants from '../../constants/indexConstants'
+import { intl } from '../../locales/intl'
+import s from '../../locales/strings.js'
+import FullScreenLoader from '../../modules/FioRequest/components/FullScreenLoader'
+import SwipeListView from '../../modules/FioRequest/components/SwipeListView'
+import { isRejectedFioRequest, isSentFioRequest } from '../../modules/FioRequest/util'
+import T from '../../modules/UI/components/FormattedText/index'
+import { styles as requestListStyles } from '../../styles/scenes/FioRequestListStyle'
```

Comment:
For the style of the request row, I would recommend importing styles that are used for the transaction row

`import styles, { styles as styleRaw } from '../../styles/scenes/TransactionListStyle'

That would save code and also ensure consistency

Reply:
yeah, it was done in previous commit - https://github.com/EdgeApp/edge-react-gui/pull/1889/commits/5f081edc3c5d1a7b86c2303dd5e3848d18df69fd, all unneeded was removed, other `requestListStyles ` is for other purposes.